src/include/sof/audio/component.h

```diff
@@ -218,9 +218,6 @@ struct comp_ops {
 	int (*position)(struct comp_dev *dev,
 		struct sof_ipc_stream_posn *posn);
 
-	/** cache operation on component data */
-	void (*cache)(struct comp_dev *dev, int cmd);
```

Comment:
What happens when you have multiple connected pipelines and some can run on different cores? Where is that managed?

Reply:
@paulstelian97 It's not yet supported and there are checks in ipc which forbid this kind of connection.

Reply:
Ah, I thought this was possible but maybe I don't remember the details I have seen a while ago. Alright.